### PR TITLE
Turn ColorCSSVariablesUpdater from demo into story

### PR DESCRIPTION
**Pull request by xiongmao86**
Turn the `ColorCSSVariablesUpdater` from demo into story.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+  title: 'services/ColorCSSVariablesUpdater',
 }
 
-export const Default = defineComponent({
-  components: { ColorCSSVariablesUpdater },
-  template: '<ColorCSSVariablesUpdater/>',
+export const Default = () => ({
+  components: { VaChip, VaDivider, VaButton },
+  methods: {
+    change (color) {
```
Comment (m0ksem) [inline packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts:11]: Would be better to have it in script setup. Even though it doesn't produce any memory leaks or other issues it feels weird for me to use hooks in options api.

File: packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+      useColors().setColors({ danger: color })
+    },
+  },
+  template: `
+    <div class="mb-6">
+      <a class="title title--va-danger">
```
Comment (m0ksem) [inline packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts:17]: We test here css variables, I don't remember what is title class and where can I find it... How about we actually use `var(--va-danger)` here and simplify this story?

File: packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
+        This is title danger color
+      </a>
+      <va-chip color="danger">This is vue component</va-chip>
+    </div>
+
+    <va-divider />
```
Comment (m0ksem) [inline packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts:23]: The style guide for stories is kind of new, I'm not really into it right now, but va-diivder feels redundant here.

**Review by m0ksem — CHANGES_REQUESTED** [review]

File: packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts [hunk 4]
```diff
@@ -1,12 +1,36 @@
...
+
+    return {
+      change,
+    }
+  },
+  template: `
+    <div class="mb-6">
+      <a class="var(--va-danger)">
+        This is title danger color
+      </a>
```
Comment (xiongmao86) [inline packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts:24]: Hi, @m0ksem, I have changed code according to your comments. I have take another look, in storybook this anchor doesn't show the danger color. I Have no idea why... :(

Comment (xiongmao86) [reply packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts]: It is a stupid mistake, I reflect on it this morning, turns out I am using class but not style.

File: packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts [hunk 5]
```diff
@@ -1,12 +1,36 @@
...
+    return {
+      change,
+    }
+  },
+  template: `
+    <div class="mb-6">
+      <a style="color: var(--va-danger)">
+        This is title danger color
+      </a>
+      <va-chip color="danger">This is vue component</va-chip>
```
Comment (m0ksem) [inline packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts:25]: Also, this story tests two cases right now: css variable and how component reacts to color config update. Should only test css variables here and test second case somewhere else. (Idk where).

```suggestion
```

File: packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts [hunk 6]
```diff
@@ -1,12 +1,36 @@
...
+    </div>
+
+    Change danger color to:
+    <va-button @click="change('#4ab2e3')" color="#4ab2e3">
+      danger = blue
```
Comment (m0ksem) [inline packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts:30]:
```suggestion
      blue (#4ab2e3)
```

File: packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts [hunk 7]
```diff
@@ -1,12 +1,36 @@
...
+    </va-button>
+    <va-button @click="change('#e34b4a')" color="#e34b4a">
+      danger = red
```
Comment (m0ksem) [inline packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts:33]: We were repeating danger = in text above and button text

```suggestion
      red (#e34b4a)
```

**Comment by m0ksem** [comment]
Thanks! Story looks much better now!

**Pull request merged**